ambry-clustermap/src/main/java/com/github/ambry/clustermap/PropertyStoreToDataNodeConfigAdapter.java

```diff
@@ -47,6 +47,7 @@
  */
 class PropertyStoreToDataNodeConfigAdapter implements DataNodeConfigSource {
   static final String CONFIG_PATH = "/DataNodeConfigs";
+  static final String SEPARATOR = "/";
```

Comment:
File.separator

Reply:
Since the is a path separator used in zookeeper nodes and is always `/ `, I am wondering if it is better to not use File.separator (which could be `\\` on windows systems although we use only linux).

Reply:
ok